autoload/maktaba/extension.vim

```diff
@@ -0,0 +1,159 @@
+""
+" @dict ExtensionRegistry
+" A registry for the extensions used a single plugin.  Extensions are
+" dictionaries with a plugin-specific interpretation
+"
+" Plugins should use |Plugin.GetExtensionRegistry| to gain access to their own
```

Comment:
Oh, right. `@function(maktaba#plugin#GetExtensionRegistry)` will get it. It should be translated to the dict spelling in the docs.